// Copyright 2015 The gRPC Authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.example.android.tflitecamerademo";
option java_outer_classname = "ImageClassifierProtoc";
option objc_class_prefix = "RTG";

package tflite;

// Interface exported by the server.
service ImageClass {
    // A simple RPC.
    //
    // Obtains the feature at a given position.
    //
    // A feature with an empty name is returned if there's no feature at the given
    // position.
    rpc getStage(GrpcRequest) returns (GrpcReply) {}


}

// Points are represented as latitude-longitude pairs in the E7 representation
// (degrees multiplied by 10**7 and rounded to the nearest integer).
// Latitudes should be in the range +/- 90 degrees and longitude should be in
// the range +/- 180 degrees (inclusive).
// The request message containing the string to be labeled
message GrpcRequest {
    //string name = 1;
    bytes image = 1;
    // repeated int32 image = 1 [packed=true];





}

// The response message containing the greetings
message GrpcReply {
    string stage = 1;
    int32 stageAcc = 2;



}